charts/rasa-x/values.yaml

```diff
@@ -99,6 +99,10 @@ rasax:
     # annotations for the service
     annotations: {}
 
+  # additional labels can be used to map organizational structures onto system objects
+  # https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+  labels: {}
```

Comment:
I'd move these fields to the `global` section and change the name to `additionalDeploymentLabels`. The `labels` field in the `rasax` section implies that extra labels are applied only for the rasa-x deployment.